output_template/python/run.py

```diff
@@ -148,6 +162,61 @@ def run_prediction(input_image, model, config_file, max_percent_incorrect_values
     _save_images(output_dir, filename_images)
 
 
+def run_prediction_with_bench(input_image, model, config_file, max_percent_incorrect_values=0.1, trial=1):
```

Comment:
`run_prediction_with_bench` and `run_prediction` have similar lines.
Is it possible to unity?

Reply:
Ya, I think so, let me try

Reply:
I wonder that making benchmark as a default behavior of run_prediction would be fine. How do you think?

Reply:
@iizukak

Reply:
@kchygoe Yeah. I think so too.